src/doc/online_decoding.dox

```diff
@@ -438,6 +438,70 @@ and downloadable models that can be used with online nnet3 decoding, please
 see http://kaldi-asr.org/models.html (the first model there, the ASPIRE model,
 includes instructions in a README file).
 
+\subsection online_decoding_nnet3_net TCP server for nnet3 online decoding
+
+The program to run the TCP sever is online2-net-nnet3-latgen-faster located in the
```

Comment:
Seems like we missed one :)

online2-net-nnet3-latgen-faster -> online2-tcp-nnet3-decode-faster

Also let's rename the subsection to online_decoding_nnet3_tcp.

Reply:
fixed